src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -2718,17 +2718,41 @@ paths:
       tags:
       - FeatureTogglesEndpoint
   /api/feedback:
+    get:
+      description: "\n        List all issues details.\n        "
+      parameters: []
+      responses:
+        '200':
+          content:
+            application/vnd.ceph.api.v1.0+json:
```

Comment:
@avanthakkar @epuertat AFAIK our current agreement is to mark as esperimental all new endpoints that are likely to be subject to changes (like these ones).